Website/src/rendering/src/interfaces/Timeslot.tsx

```diff
@@ -0,0 +1,7 @@
+import { Field } from '@sitecore-jss/sitecore-jss-nextjs';
```

Comment:
Should this file by a `.ts` file instead of a `.tsx` file? It does not output JSX.

Reply:
This does not seem to have been resolved.